enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -100,6 +98,15 @@ describe("InteractiveQuestion", () => {
     ).toBeInTheDocument();
   });
 
+  it("should not render an error if a question isn't found before the question loaded", async () => {
```

Comment:
You can verify this test by running this against the previous version of the code and it should fail.